### DISCUSSION TITLE
Find Vercel Account Info After GitHub Auth Switches

### DISCUSSION THREAD

**Discussion by bottbott** [vercel/community#1468] [Help]
Hey there, 

Feel rather foolish about this, but a few months ago I associated my GitHub account with a new account I set up on Vercel, and now I can't seem to login to my old account credentials (which I honestly thought was just setup through GitHub). 

Does anyone have any ideas for how I could track that down? I'm able to push commits to the repo still and the Vercel integration automatically pushes them to the old account, but can't find any other clues.

**Comment by amyegan** [comment]
Since said you're unable to [log in](https://vercel.com/login?next=/dashboard) with your old account credentials, do you mean that  you are using the "Continue with GitHub" button or are you trying to log in with an email address?

![Screenshot 2023-02-07 at 3 13 45 PM](https://user-images.githubusercontent.com/8247105/217355102-2a97c460-faba-42b6-bb41-1d5097fba4cf.png)

If you are still having trouble, you can open a new support case by emailing [registration@vercel.com](mailto:registration@vercel.com) with details of the error you're seeing. That way you can safely discuss account details via email instead of in public here 😄

**Comment by bottbott** [comment]
Logged a ticket after this post! It looks like my associated email fell out of sync when I updated my GitHub email, and now Vercel is still tied to the original email on my GitHub. I auth'd in the whole time with GitHub so it's a bit ackward and don't have access to the old email anymore. We'll see how the support chat goes.
